File: CoffeeShop_Web/CoffeeShop/src/main/java/com/example/CoffeeShop/entity/MenuItem.java
package com.example.CoffeeShop.entity;

import com.google.firebase.database.PropertyName;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class MenuItem {
    private String id;
    private String name;
    private double price;
    private String status;

    private String categoryId;
    private String imageUrl;
    private String createdAt;

    @PropertyName("category_id")
    public String getCategoryId() {
        return categoryId;
    }

    @PropertyName("category_id")
    public void setCategoryId(String categoryId) {
        this.categoryId = categoryId;
    }

    @PropertyName("image_url")
    public String getImageUrl() {
        return imageUrl;
    }

    @PropertyName("image_url")
    public void setImageUrl(String imageUrl) {
        this.imageUrl = imageUrl;
    }

    @PropertyName("createdAt")
    public String getCreatedAt() {
        return createdAt;
    }

    @PropertyName("createdAt")
    public void setCreatedAt(String createdAt) {
        this.createdAt = createdAt;
    }
    
}

File: CoffeeShop_Moblie/app/src/main/java/com/example/coffeeshopmobile/data/model/MenuItem.kt
package com.example.coffeeshopmobile.data.model

import com.google.firebase.database.IgnoreExtraProperties
import com.google.firebase.database.PropertyName

@IgnoreExtraProperties
data class MenuItem(
    var id: String = "",
    var name: String = "",
    var price: Double = 0.0,
    var status: String = "available", // available (Còn hàng), unavailable (Hết hàng)

    @get:PropertyName("category_id")
    @set:PropertyName("category_id")
    var categoryId: String = "",

    @get:PropertyName("image_url")
    @set:PropertyName("image_url")
    var imageUrl: String? = null,

    var createdAt: String? = null
)